src/cpu_profile.cc

```diff
@@ -12,93 +12,95 @@ namespace nodex {
   using v8::Object;
   using v8::ObjectTemplate;
   using v8::FunctionTemplate;
-  using v8::Persistent;
   using v8::String;
   using v8::Function;
   using v8::Value;
 
-  Persistent<ObjectTemplate> Profile::profile_template_;
-  Persistent<Array> Profile::profiles;
+  Nan::Persistent<ObjectTemplate> Profile::profile_template_;
+  Nan::Persistent<Array> Profile::profiles;
   uint32_t Profile::uid_counter = 0;
-  
+
+  NAN_METHOD(Profile_EmptyMethod) {
+  }
+
   void Profile::Initialize () {
-    NanScope();
-    
-    Local<FunctionTemplate> f = NanNew<FunctionTemplate>();
+    Nan::HandleScope scope;
+
+    Local<FunctionTemplate> f = Nan::New<FunctionTemplate>(Profile_EmptyMethod);
```

Comment:
You're right. It probably is a NAN bug. The code is new to me and it does ... _things_, so I did not spot it on first glance. @kkoopa, if you can't find it either let me know. Then I'll dig into it.